addOns/paramminer/src/test/java/org/zaproxy/addon/paramminer/UrlUtilsUnitTest.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2022 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.addon.paramminer;
+
+import static fi.iki.elonen.NanoHTTPD.newFixedLengthResponse;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import fi.iki.elonen.NanoHTTPD.IHTTPSession;
+import fi.iki.elonen.NanoHTTPD.Response;
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.parosproxy.paros.network.HttpMessage;
+import org.parosproxy.paros.network.HttpRequestHeader;
+import org.zaproxy.zap.testutils.NanoServerHandler;
+import org.zaproxy.zap.testutils.TestUtils;
+
+class UrlUtilsUnitTest extends TestUtils {
+
+    @BeforeEach
+    void init() throws IOException {
+        startServer();
+    }
+
+    @AfterEach
+    void teardown() {
+        stopServer();
+    }
+
+    @Test
+    void shouldLoadFileWithoutErrors() throws IOException {
+        String path =
+                new File(
+                                "./src/main/resources/org/zaproxy/addon/paramminer/resources/wordlists/smalllist.txt")
```

Comment:
Should use a relative path, most likely you'll have to use `Class#getResource` or `#getResourceAsStream` if you want to keep them in the add-on.

Reply:
Uhh, the relative path will then have multiple "../../../ ----" then :)

Reply:
That's fine.

Reply:
You can do something like:
https://github.com/zaproxy/zap-extensions/blob/544363964beb1fca7d12c8fab94af19277e7d11e/addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/HiddenFilesScanRule.java#L331-L332

Reply:
Okay so remove the wordlist from resources to zapHomeFiles dir?

Reply:
I mean make a zapHomeFiles dir and then do it?

Reply:
You can keep it in the resources dir but you need to use one of the mentioned methods, when running in ZAP the file will not be in the file system but inside the add-on.

Reply:
zapHomeFiles will be better I guess :)

Reply:
If it's in zap home then users can edit it if they have a need.